glyphs-reader/src/font.rs

```diff
@@ -304,7 +304,7 @@ impl Layer {
 
     pub(crate) fn components(&self) -> impl Iterator<Item = &Component> + '_ {
         self.shapes.iter().filter_map(|shape| match shape {
-            Shape::Path(_) => None,
+            Shape::Path(..) => None,
```

Comment:
this change is left over from an earlier version?

Reply:
Ty, fixed